.github/workflows/ci.yml

```diff
@@ -99,7 +99,7 @@ jobs:
     needs: filter-changes
     if: ${{ needs.filter-changes.outputs.nondocchanges == 'true' }}
     strategy:
-      fail-fast: true
+      fail-fast: false
```

Comment:
Why the change? Especially for the `build` job failing faster should be better? 🤔

Reply:
A temporary change to more easily see the effects on all matrix entries. Otherwise without this if a change fails for one matrix entry you don't get to find out if it would have worked on any other entries.